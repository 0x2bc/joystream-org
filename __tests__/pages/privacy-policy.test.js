import React from 'react';
import renderer from 'react-test-renderer';

import PrivacyPolicyPage from '../../src/pages/privacy-policy';

describe('PrivacyPolicyPage page', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<PrivacyPolicyPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
