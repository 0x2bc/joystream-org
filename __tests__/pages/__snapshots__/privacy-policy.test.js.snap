// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPolicyPage page renders correctly 1`] = `
<div>
  <nav
    className="Navbar Navbar--hidden"
  >
    <div
      className="Navbar__container"
    >
      <a
        className="Link"
        href="/"
      >
        <icon-mock
          className="Navbar__logo"
        />
      </a>
      <div
        className="Navbar__links"
      >
        <a
          className="Navbar__link Navbar__link--highlighted Link Link--highlighted"
          href="https://www.jsgenesis.com/#openings"
          label="We are hiring!"
        >
          We are hiring!
        </a>
        <a
          className="Navbar__link Link"
          href="https://blog.joystream.org/"
          label="Blog"
        >
          Blog
        </a>
        <a
          className="Navbar__link Link"
          href="https://blog.joystream.org/manifesto/"
          label="Manifesto"
        >
          Manifesto
        </a>
        <a
          className="Navbar__link Link"
          href="https://github.com/Joystream/whitepaper/blob/master/paper.pdf"
          label="Whitepaper"
        >
          Whitepaper
        </a>
        <a
          className="Navbar__link Link"
          href="/roles"
          label="Roles"
        >
          Roles
        </a>
        <a
          className="Navbar__link Link"
          href="https://github.com/Joystream/joystream"
          label="Repository"
        >
          Repository
        </a>
      </div>
      <div
        className="Navbar__trigger"
        onClick={[Function]}
        role="presentation"
      />
    </div>
  </nav>
  <section
    className="Hero"
  >
    <div
      className="Hero__container"
    >
      <div
        className="Hero__content"
      >
        <h1
          className="Hero__title"
        >
          Privacy Policy and Cookies
        </h1>
        <div
          className="Hero__description"
        >
          <p
            className="PrivacyPolicyPage__hero-paragraph"
          >
            <strong>
              Jsgenesis values your privacy.
            </strong>
            <br />
            <br />
            This Privacy Policy ("Privacy Policy") and Cookie Policy ("Cookie Policy") explains how Jsgenesis AS ("Jsgenesis", "Company", "We", "Us", "Our") collect and use data and information when you ("User) use on or any of the Joystream products, developed in the GitHub organization
             
            <a
              className="Link Link--highlighted"
              href="https://github.com/JoyStream"
            >
              Joystream
            </a>
            . These products (collectively "Software") include, but are not limited to, all pages under the
             
            <a
              className="Link Link--highlighted"
              href="https://www.joystream.org/"
            >
              joystream.org
            </a>
             
            domain ("Website"), the
             
            <a
              className="Link Link--highlighted"
              href="https://github.com/Joystream/substrate-node-joystream"
            >
              Joyful node
            </a>
             
            ("Full Node"), the
             
            <a
              className="Link Link--highlighted"
              href="https://github.com/Joystream/storage-node-joystream"
            >
              Colossus Storage Node
            </a>
             
            ("Storage node"), and the Pioneer User Interface, either
             
            <a
              className="Link Link--highlighted"
              href="https://github.com/Joystream/apps"
            >
              self hosted
            </a>
             
            or
             
            <a
              className="Link Link--highlighted"
              href="http://testnet.joystream.org/"
            >
              hosted by Us
            </a>
             
            ("App").
          </p>
        </div>
      </div>
    </div>
  </section>
  <div
    className="LayoutWrapper"
  >
    <div
      className="LayoutWrapper__container"
    >
      <section
        className="PolicyWrapper PolicyWrapper--left"
      >
        <div
          className="PolicyWrapper__header"
        >
          <h2
            className="PolicyWrapper__title"
          >
            Relevant to the Privacy Policy and Cookie Policy are the following terms:
          </h2>
          
        </div>
        <ul>
          <li
            className="PrivacyPolicyPage__list-item"
          >
            The term 
            <strong>
              "Blockchain"
            </strong>
             refers to the blockchain(s) assembled by the Full Node.
          </li>
          <li
            className="PrivacyPolicyPage__list-item"
          >
            The term 
            <strong>
              "Content"
            </strong>
             refers to media files accessible through our Software.
          </li>
          <li
            className="PrivacyPolicyPage__list-item"
          >
            The term 
            <strong>
              "Keys"
            </strong>
             refers to a private/public cryptographic keypair, that Users can generate in order to write (and decrypt data) on the Blockchain.
          </li>
          <li
            className="PrivacyPolicyPage__list-item"
          >
            The term 
            <strong>
              "Membership"
            </strong>
             refers to tying your Keys to a public profile, allowing users to access Content and interact with the Blockchain.
          </li>
          <li
            className="PrivacyPolicyPage__list-item"
          >
            The term 
            <strong>
              "Memo"
            </strong>
             refers to a markdown enabled text field, where users can input data tied to their Keys.
          </li>
        </ul>
      </section>
      <section
        className="PolicyWrapper"
      >
        <div
          className="PolicyWrapper__header"
        >
          <h2
            className="PolicyWrapper__title"
          >
            Privacy Policy
          </h2>
          <p
            className="PolicyWrapper__subtitle"
          >
            Last updated on the 17th of April 2019
          </p>
        </div>
        <div
          className="ColumnsLayout ColumnsLayout--large"
          data-columns={2}
        >
          <div
            className="PolicyItem "
          >
            <p
              className="PolicyItem__title"
            >
              1. Agreement to the Policy
            </p>
            <div
              className="PolicyItem__content"
            >
              By using any of Our Software, the User are accepting this Privacy Policy. If you are acting on behalf of another company or an employer, you must have the rights to act on their behalf. The Privacy Policy is not extended to any of our newsletters, where Users are bound by the
               
              <a
                className="Link"
                href="https://mailchimp.com/legal/privacy/"
              >
                privacy policy
              </a>
               of
               
              <a
                className="Link"
                href="https://mailchimp.com/"
              >
                Mailchimp
              </a>
              <br />
              ‍
              <br />
              The Privacy Policy does not apply to any other third party services including, but not limited to, applications, websites, tools or software, even if accessible through links or guides in our Software.
            </div>
          </div>
          <div
            className="PolicyItem "
          >
            <p
              className="PolicyItem__title"
            >
              2. Changes to Policy
            </p>
            <div
              className="PolicyItem__content"
            >
              This Privacy Policy may be changed at the sole discretion of Company. If any material changes are made,
      the User will be notified in the Service that is used. Note that adding new products to be included in the term
      Software, e.g. a new User facing product replacing the App or a new tool for uploading Content, is not considered
      material as it will not affect Users unless they adopt the new product. Changing software names,
      terminology used in this Privacy Policy, and changing link locations are also examples of non-material changes.
            </div>
          </div>
          <div
            className="PolicyItem "
          >
            <p
              className="PolicyItem__title"
            >
              3. Information Collected
            </p>
            <div
              className="PolicyItem__content"
            >
              All data written to the Blockchain, is implicitly collected not only by Company, but also anyone else in the world that is running the Full Node locally, or accessed via the App or a third party. This includes, but is not limited to, Content hashes, Membership profile, Memo field, and any other way a User can record data on the Blockchain. ‍
              <br />
              <br />
              When using the 
              <a
                className="Link"
                href="https://testnet.joystream.org/faucet"
              >
                faucet
              </a>
               subpage of the Website, Company will record the IP address behind every new request for tokens. This data will be deleted every 14 days.
            </div>
          </div>
          <div
            className="PolicyItem "
          >
            
            <div
              className="PolicyItem__content PolicyItem__content--padded"
            >
              Company uses 
              <a
                className="Link"
                href="https://marketingplatform.google.com/about/analytics/"
              >
                Google Analytics
              </a>
              , with IP anonymization, to collect statistics on Website and the version of App hosted by us. All customisable data sharing settings are turned off to improve the privacy of Users. ‍
              <br />
              ‍
              <br />
              <strong>
                Company will not sell your data for advertising, or other purposes.
              </strong>
            </div>
          </div>
        </div>
      </section>
    </div>
  </div>
  <div
    className="LayoutWrapper LayoutWrapper--dark"
  >
    <div
      className="LayoutWrapper__container"
    >
      <section
        className="PolicyWrapper"
      >
        <div
          className="PolicyWrapper__header"
        >
          <h2
            className="PolicyWrapper__title"
          >
            Cookies Policy
          </h2>
          <p
            className="PolicyWrapper__subtitle"
          >
            Last updated on the 17th of April 2019
          </p>
        </div>
        <div
          className="ColumnsLayout ColumnsLayout--large"
          data-columns={2}
        >
          <div
            className="PolicyItem "
          >
            <p
              className="PolicyItem__title"
            >
              1. What are Cookies?
            </p>
            <div
              className="PolicyItem__content"
            >
              Cookies are small pieces of text sent by your web browser by a website you visit. A cookie file is stored in your web browser and allows the Service or a third-party to recognise you and make your next visit easier and the Service more useful to you. ‍
              <br />
              ‍
              <br />
              Cookies can be persistent or session cookies.
            </div>
          </div>
          <div
            className="PolicyItem "
          >
            <p
              className="PolicyItem__title"
            >
              2. How we use Cookies
            </p>
            <div
              className="PolicyItem__content"
            >
              We use cookies for the following purposes our Service:
              <br />
              <br />
              <ul>
                <li>
                  Provide Analytics
                </li>
                <li>
                  Store preferences
                </li>
                <li>
                  Persistant local storage of Keys and Membership.
                </li>
              </ul>
            </div>
          </div>
          <div
            className="PolicyItem "
          >
            <p
              className="PolicyItem__title"
            >
              3. Third-party Cookies
            </p>
            <div
              className="PolicyItem__content"
            >
              In addition to our own cookies, we also use various third-party cookies to report usage statistics of the Service, deliver advertisements on and through the Service, and so on. They include:
              <br />
              <br />
              <ul>
                <li>
                  <strong>
                    Google Analytics
                  </strong>
                </li>
                <li>
                  <strong>
                    Mailchimp
                  </strong>
                   (Only when signing up for any of our newsletters)
                </li>
                <li>
                  <strong>
                    Godaddy
                  </strong>
                </li>
              </ul>
              <br />
              Please see Item 3. of the Privacy Policy for more information on the extent of these providers.
            </div>
          </div>
          <div
            className="PolicyItem "
          >
            <p
              className="PolicyItem__title"
            >
              Your Regarding Cookies
            </p>
            <div
              className="PolicyItem__content"
            >
              If you would like to delete cookies or instruct your web browser to delete or refuse cookies, please visit the help pages of your web browser.
              <br />
              ‍
              <br />
              Please note, however, that if you delete cookies or refuse to accept them, you might not be able to use all of the features we offer, you may not be able to store your preferences, and some of our pages might not display properly.
            </div>
          </div>
        </div>
      </section>
      <div
        className="PrivacyPolicyPage__links"
      >
        <a
          className="Link"
          href="http://www.allaboutcookies.org/"
        >
          AllAboutCookies
        </a>
        <a
          className="Link"
          href="http://www.networkadvertising.org/"
        >
          Network Advertising Initiative
        </a>
      </div>
    </div>
  </div>
  <div
    className="CookiesNotice"
  >
    <div
      className="CookiesNotice__container"
    >
      <div
        className="CookiesNotice__content"
      >
        <p>
          This site uses cookies. By continuing to browse the site, you are agreeing to our use of cookies.
        </p>
        <p>
          <a
            className="CookiesNotice__link Link"
            href="/privacy-policy"
          >
            Find out more ›
          </a>
        </p>
      </div>
      <button
        className="CookiesNotice__accept Button Button--secondary"
        onClick={[Function]}
        type="button"
      >
        Accept
      </button>
    </div>
  </div>
  <footer
    className="Footer"
  >
    <p
      className="Footer__header"
    >
      Get notified
    </p>
    <a
      className="Footer__button Button Button--small Button--light"
      href="http://joystream.us11.list-manage.com/subscribe/post?u=932de577aec9616d4516b4e0f&id=459ba8d1da"
      onClick={null}
    >
      Join the newsletter
    </a>
    <div
      className="Footer__socials"
    >
      <a
        className="Footer__social-wrapper"
        href="https://twitter.com/JoyStreamApp"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://github.com/Joystream"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://chat.joystream.org/home"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://t.me/joinchat/CNyeUxHD9H56m3e_44hXIA"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
    </div>
    <div
      className="Footer__links"
    >
      <a
        className="Footer__link Footer__link--highlighted Link Link--highlighted"
        href="https://www.jsgenesis.com/#openings"
        label="We are hiring!"
      >
        We are hiring!
      </a>
      <a
        className="Footer__link Link"
        href="https://blog.joystream.org/"
        label="Blog"
      >
        Blog
      </a>
      <a
        className="Footer__link Link"
        href="https://blog.joystream.org/manifesto/"
        label="Manifesto"
      >
        Manifesto
      </a>
      <a
        className="Footer__link Link"
        href="https://github.com/Joystream/joystream"
        label="Repository"
      >
        Repository
      </a>
    </div>
  </footer>
</div>
`;
